app/shared/productTypes.tsx

```diff
@@ -443,7 +446,8 @@ export const ProductTypes: { [productKey in ProductTypeKeys]: ProductType } = {
       }
     },
     delivery: {
-      showAddress: showDeliveryAddressCheck
+      showAddress: showDeliveryAddressCheck,
+      enableDeliveryInstructionsUpdate: true
```

Comment:
think we need to check if Delivery Instructions are relevant for Voucher actually??

Reply:
@JessL123

Reply:
@JessL123 said `Yup`